gravitee-apim-console-webui/src/portal/navigation-items/portal-navigation-items.component.ts

```diff
@@ -342,6 +342,33 @@ export class PortalNavigationItemsComponent {
       .subscribe();
   }
 
+  onDeleteSection(node: SectionNode): void {
+    this.portalNavigationItemsService
+      .deleteNavigationItem(node.id)
+      .pipe(
+        tap(() => {
+          const currentNavId = this.navId();
+          if (currentNavId === node.id) {
+            this.router
+              .navigate(['.'], {
+                relativeTo: this.activatedRoute,
+                queryParams: { navId: null },
+                queryParamsHandling: 'merge',
+              })
+              .catch(() => this.snackBarService.error('Failed to update selection after deletion'));
+          }
+          this.refreshMenuList.next(1);
+          this.snackBarService.success(`Section "${node.label}" deleted`);
```

Comment:
Let's keep it a bit more generic: 
```suggestion
          this.snackBarService.success(`Navigation item "${node.label}" deleted`);
```